### PR TITLE
Fixed bug for missing commands in ubuntu/debian and centos

### LINKED ISSUE #2761: Agent Deployment missing instructions to deploy Linux Agents
| Wazuh | Elastic | Rev |
| ----- | ------- | --- |
| 4.0.3   | 7.9.1     | x |

**Description**
Agents Deployment instructions are missing the commands required to enable the service.

**Steps to reproduce**

1. Go to 'Wazuh App -> Agents -> Deploy new agent`
2. Follow the instructions to deploy a new agent.
3. Restart the agent host and the service won't start.

**Screenshots**
Deploy new agent

**Additional context**
If you follow the instructions from the documentation it works as intended:
https://documentation.wazuh.com/4.0/installation-guide/wazuh-agent/wazuh_agent_package_linux.html

**Systemd**
```
systemctl daemon-reload
systemctl enable wazuh-agent
systemctl start wazuh-agent
```

**SysV Init**
RPM based OS:
```
chkconfig --add wazuh-agent
service wazuh-agent start
```
Debian based OS:
```
update-rc.d wazuh-agent defaults 95 10
service wazuh-agent start
```

### PR DESCRIPTION
**Pull request by CPAlejandro**
Hi guys, we've fixed the following bug: missing steps for rebooting the agent to reboot with us. We have added one more step for Redhat / centOS and Ubuntu / debian to add the missing commands.
Closes #2761

### CODE REVIEW

**Comment by Desvelao** [comment]
I tested the PR.

Some screenshots:
Centos:
  - Systemd
image

- SysV
image

Debian/Ubuntu
  - Systemd
image
 - SysV
image

**Review by Desvelao — APPROVED** [review]
LGTM!

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ All notable changes to the Wazuh app project will be documented in this file.
 
 - Fixed Wazuh menu and agent menu for Solaris agents [#2773](https://github.com/wazuh/wazuh-kibana-app/issues/2773) [#2725](https://github.com/wazuh/wazuh-kibana-app/issues/2725)
 - Fixed wrong shards and replicas for statistics indices and also fixed wrong prefix for monitoring indices [#2732](https://github.com/wazuh/wazuh-kibana-app/issues/2732)
+- Fixed bug for missing commands in ubuntu/debian and centos [#2786](https://github.com/wazuh/wazuh-kibana-app/issues/2773) [#2725](https://github.com/wazuh/wazuh-kibana-app/issues/2786)
```
Comment (Desvelao) [inline CHANGELOG.md]: The issue link should be `https://github.com/wazuh/wazuh-kibana-app/issues/2786`
```
[#2786](https://github.com/wazuh/wazuh-kibana-app/issues/2773)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -9,6 +9,9 @@ All notable changes to the Wazuh app project will be documented in this file.
 - Fixed Wazuh menu and agent menu for Solaris agents [#2773](https://github.com/wazuh/wazuh-kibana-app/issues/2773) [#2725](https://github.com/wazuh/wazuh-kibana-app/issues/2725)
 - Fixed wrong shards and replicas for statistics indices and also fixed wrong prefix for monitoring indices [#2732](https://github.com/wazuh/wazuh-kibana-app/issues/2732)
 - Report's creation dates set to 1970-01-01T00:00:00.000Z [#2772](https://github.com/wazuh/wazuh-kibana-app/issues/2772)
+- Fixed bug for missing commands in ubuntu/debian and centos [#2725](https://github.com/wazuh/wazuh-kibana-app/issues/2786)
```
Comment (Desvelao) [inline CHANGELOG.md:12]: The link text should be `#2786` and not `#2725` in:
```
[#2725](https://github.com/wazuh/wazuh-kibana-app/issues/2786)
```

File: public/controllers/agent/components/register-agent.js [hunk 3]
```diff
@@ -307,17 +346,25 @@ export class RegisterAgent extends Component {
...
-          ...(!this.state.selectedArchitecture ? ['OS architecture'] : [])
+          ...(!this.state.selectedArchitecture ? ['OS architecture'] : []),
         ];
       default:
         return [];
     }
   };
 
+  checkNeedSystemdOrSysV(os){
+    if(os === 'rpm'){
```
Comment (frankeros) [inline public/controllers/agent/components/register-agent.js:361]: you could simply do
`return os === 'rpm';`

File: CHANGELOG.md (hunk 2 above)
Comment (frankeros) [reply CHANGELOG.md]: It's ok the first part [title] [issue id], #2725 is the issue that was fixed, but in the link should be the url to the same issue id #2725

**Review by frankeros — APPROVED** [review]
LGTM! Great Job :tada:

**Pull request merged**